lib/node_modules/@stdlib/ndarray/flatten/README.md

```diff
@@ -0,0 +1,127 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# flatten
+
+> Return a flattened copy of an input [`ndarray`][@stdlib/ndarray/ctor].
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var flatten = require( '@stdlib/ndarray/flatten' );
+```
+
+#### flatten( x, depth )
+
+Returns a flattened copy of an input [`ndarray`][@stdlib/ndarray/ctor].
+
+```javascript
+var array = require( '@stdlib/ndarray/array' );
+var ndarray2array = require( '@stdlib/ndarray/to-array' );
+
+// Create an input ndarray:
+var x = array( [ [ [ 1.0, 2.0 ] ], [ [ 3.0, 4.0 ] ], [ [ 5.0, 6.0 ] ] ] );
+// returns <ndarray>
+
+var y = flatten( x, 2 );
+// returns <ndarray>
+
+var arr = ndarray2array( y );
+// returns [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ]
+```
+
+The function accepts the following arguments:
+
+-   **x**: input [`ndarray`][@stdlib/ndarray/ctor].
+-   **depth**: a non-negative integer specifying the number of input [`ndarray`][@stdlib/ndarray/ctor] dimensions to flatten.
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   The provided input [`ndarray`][@stdlib/ndarray/ctor] **must** have more than one-dimension.
```

Comment:
I don't think this is correct. Flattening a one-dimensional ndarray should be a no-op, but, as we are always returning a new array, providing a one-dimensional array should just be a straightforward copy.

Reply:
The more interesting question is whether we should support zero-dimensional ndarrays. I suggest seeing what NumPy allows.